Remote Habits/View/RHDashboardViewController.swift

```diff
@@ -73,15 +90,15 @@ extension RHDashboardViewController : UITableViewDelegate {
     func tableView(_ tableView: UITableView, viewForHeaderInSection section: Int) -> UIView? {
         let headerView = UIView.init(frame: CGRect.init(x: 0, y: 0, width: tableView.frame.width, height: 97))
         headerView.backgroundColor = self.view.backgroundColor
-        let headerData = dashboardData[section].first?.key
+        let headerData = dashboardHeaders[section]
         let label = UILabel()
         label.frame = CGRect.init(x: 16, y: 5, width: headerView.frame.width-20, height: headerView.frame.height-10)
         if section != 0 {
             label.frame = CGRect.init(x: 16, y: 60, width: headerView.frame.width-20, height: 26)
         }
         
-        label.text = headerData?.headerTitle
-        label.font = UIFont(name: headerData?.titleFontName ?? RHConstants.kEmptyValue, size: CGFloat(headerData?.titleFontSize ?? 17))
+        label.text = headerData.headerTitle ?? RHConstants.kEmptyValue
```

Comment:
```suggestion
        label.text = headerData.headerTitle ?? ""
```

I think this reads better. I don't think we need a constant for empty string as it makes the code harder to read and doesn't add much value.